src/features/settings/gestures/SwipeSettings.tsx

```diff
@@ -0,0 +1,232 @@
+import { IonActionSheet, IonLabel, IonList, IonToggle } from "@ionic/react";
+import { InsetIonItem, ListHeader } from "../shared/formatting";
+import { useAppDispatch, useAppSelector } from "../../../store";
+import {
+  OSwipeActionPost,
+  OSwipeActionComment,
+  SwipeAction,
+  SwipeActions,
+  OSwipeActionInbox,
+  OSwipeActionAll,
+} from "../../../services/db";
+import SettingSelector from "../shared/SettingSelector";
+import {
+  setPostSwipeActionFarEnd,
+  setPostSwipeActionFarStart,
+  setPostSwipeActionEnd,
+  setPostSwipeActionStart,
+  setCommentSwipeActionFarEnd,
+  setCommentSwipeActionFarStart,
+  setCommentSwipeActionEnd,
+  setCommentSwipeActionStart,
+  setInboxSwipeActionFarEnd,
+  setInboxSwipeActionFarStart,
+  setInboxSwipeActionEnd,
+  setInboxSwipeActionStart,
+  setAllSwipesToDefault,
+  setDisableLeftSwipes,
+  setDisableRightSwipes,
+} from "./gesturesSlice";
+import { ActionCreatorWithPayload } from "@reduxjs/toolkit";
+import { Dictionary } from "lodash";
+import { useState } from "react";
+import { IonActionSheetCustomEvent, OverlayEventDetail } from "@ionic/core";
+import { ReactComponent as ShortSwipeSvg } from "./swipe_short.svg";
+import { ReactComponent as LongSwipeSvg } from "./swipe_long.svg";
+import {
+  arrowDownOutline,
+  arrowUndoOutline,
+  arrowUpOutline,
+  bookmarkOutline,
+  chevronCollapseOutline,
+  eyeOffOutline,
+  mailUnreadOutline,
+} from "ionicons/icons";
+
+export default function SwipeSettings() {
+  const [open, setOpen] = useState(false);
+  const dispatch = useAppDispatch();
+
+  const post = useAppSelector((state) => state.gestures.swipe.post);
+  const comment = useAppSelector((state) => state.gestures.swipe.comment);
+  const inbox = useAppSelector((state) => state.gestures.swipe.inbox);
+
+  const disableLeftSwipes = useAppSelector(
+    (state) => state.gestures.swipe.disableLeftSwipes
+  );
+  const disableRightSwipes = useAppSelector(
+    (state) => state.gestures.swipe.disableRightSwipes
+  );
+
+  return (
+    <>
+      <SwipeList
```

Comment:
Can you remove the chevrons on each swipe setting?

Reply:
What do you mean?

Reply:
These guys, we can just remove them for all settings that open the action sheet
![IMG_7838E59B0D07-1](https://github.com/aeharding/voyager/assets/2166114/2f56a1a2-0acc-4eff-9db3-21ae09f251f2)

Reply:
It turns out you can't super easily disable this. It turns on whenever you have an IonItem that is a button, because that is what iOS interfaces look like.

Reply:
Never mind, I figured it out. Although it looks a little weird to me; if you go into iOS settings every single menu has a chevron.

Reply:
You should be able to disable with `detail={false}` or something. This is to match Apollo